### PR TITLE
Clearer instructions and resolution of problems encountered

### PR DESCRIPTION
**Pull request by Lulutima**
Add some instructions to to get around the errors and add video for a better understanding of the instructions.

### CODE REVIEW

**Comment by Lulutima** [comment]
https://github.com/Lulutima/MagiskOnWSA/blob/main/WSA_Magisk-Install.ps1

**Review by Caleblittle649 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,9 +23,11 @@
     ![Download](https://docs.github.com/assets/images/help/repository/artifact-drop-down-updated.png)
 1. Unzip the artifact and uninstall WSA if you have an official installation or replace the pevious unzipped artifact if you have a manual installation
 1. Enable developer mode on Windows
-1. Open powershell with admin privileges and run `Add-AppxPackage -Register .\AppxManifest.xml` under the unzipped artifact directory
+1. Follow the video [https://www.youtube.com/watch?v=EFV-fM81Tzs](https://www.youtube.com/watch?v=EFV-fM81Tzs) or the next instructions :
```
Comment (yujincheng08) [inline README.md:26]: I think we can embed videos now? https://github.blog/2021-05-13-video-uploads-available-github/

File: README.md [hunk 2]
```diff
@@ -23,9 +23,11 @@
...
+1. (If you have already install WSA, uninstall with `Remove-AppxPackage MicrosoftCorporationII.WindowsSubsystemForAndroid_1.7.32815.0_x64__8wekyb3d8bbwe -AllUsers` in powershell with admin privileges)
+1. In powershell, run `Add-AppxPackage -Register .\AppxManifest.xml` under the unzipped artifact directory (`import-module appx -usewindowspowershell` if Add-AppxPackage doesn't work)
 1. Launch WSA and enable developer mode, launch the file manager, and wait until the file manager popup
-1. Run `adb connect localhost:58526` to connect to WSA and install Magisk app (the one you use to build) and launch it
+1. Run `adb connect @IP` to connect to WSA and install Magisk app `adb install magisk.apk` (the one you use to build) and launch it
```
Comment (yujincheng08) [inline README.md:30]: `@IP` is more confusing

File: README.md [hunk 3]
```diff
@@ -23,9 +23,11 @@
     ![Download](https://docs.github.com/assets/images/help/repository/artifact-drop-down-updated.png)
 1. Unzip the artifact and uninstall WSA if you have an official installation or replace the pevious unzipped artifact if you have a manual installation
 1. Enable developer mode on Windows
-1. Open powershell with admin privileges and run `Add-AppxPackage -Register .\AppxManifest.xml` under the unzipped artifact directory
+1. Follow the video [https://www.youtube.com/watch?v=EFV-fM81Tzs](https://www.youtube.com/watch?v=EFV-fM81Tzs) or the next instructions :
+1. (If you have already install WSA, uninstall with `Remove-AppxPackage MicrosoftCorporationII.WindowsSubsystemForAndroid_1.7.32815.0_x64__8wekyb3d8bbwe -AllUsers` in powershell with admin privileges)
```
Comment (yujincheng08) [inline README.md:27]: grammar: `have install`

and there's no need to uninstall it if there's an existing sideload.

File: WSA_Magisk-Install.ps1 [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        Throw "Run as an administrator !" 
+    }
+    import-module appx -usewindowspowershell
+    Remove-AppxPackage MicrosoftCorporationII.WindowsSubsystemForAndroid_1.7.32815.0_x64__8wekyb3d8bbwe -AllUsers
+    Add-AppxPackage -Register .\AppxManifest.xml
+}
+catch
+{
+    Write-Host $_.Exception.Message -ForegroundColor Red
+}
```
Comment (yujincheng08) [inline WSA_Magisk-Install.ps1:16]: tail new line

File: WSA_Magisk-Install.ps1 [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿try
```
Comment (yujincheng08) [inline WSA_Magisk-Install.ps1:1]: Is this script really done by you? I saw one here: https://github.com/LSPosed/MagiskOnWSA/issues/30#issue-1040017175

Comment (yujincheng08) [reply WSA_Magisk-Install.ps1:1]: I prefer a script that can automatically prompt UAC request

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Lulutima) [reply README.md:26]: Yes, it's better this way

File: README.md (hunk 2 above)
Comment (Lulutima) [reply README.md:30]: localhost doesn't work for me, I had to put the IP address of the wsa, present in the settings

File: README.md (hunk 3 above)
Comment (Lulutima) [reply README.md:27]: I had installed the wsa with the official installer from the store, but when I right clicked uninstall, the package was still present and the modified version with magisk did not work

File: WSA_Magisk-Install.ps1 (hunk 5 above)
Comment (Lulutima) [reply WSA_Magisk-Install.ps1:1]: Yes it's very basic, it's a good idea

**Pull request closed without merging**